package.json

```diff
@@ -16,19 +16,21 @@
     "@fontsource/roboto": "^4.5.8",
     "@mui/material": "^5.10.0",
     "@mui/styled-engine-sc": "^5.10.0",
+    "angular": "^1.8.3",
     "aws-sdk": "^2.1194.0",
     "axios": "^0.27.2",
+    "cors": "^2.8.5",
     "dotenv": "^16.0.1",
     "express": "^4.18.1",
-    "multer": "^1.4.5-lts.1",
-    "cors": "^2.8.5",
     "firebase": "^9.9.2",
+    "multer": "^1.4.5-lts.1",
     "nodemon": "^2.0.19",
     "path": "^0.12.7",
     "pg": "^8.7.3",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-firebase-hooks": "^5.0.3",
+    "react-google-login": "^5.2.2",
```

Comment:
Follow up, noticed `react-google-login` is causing an pnm install conflict because it is looking for React 16 or 17